src/convert.rs

```diff
@@ -284,7 +266,7 @@ pub fn elf_to_tbf(
     let mut writeable_flash_regions_count = 0;
 
     for s in &sections_sort {
```

Comment:
I think we should be getting rid of this loop. Or, perhaps removing everything not related to writeable flash regions, and moving it to just the wfr handling section down below.

Reply:
Could you please elaborate it further? Is this only about the rel_sections to populate the sections?